.github/workflows/build.yml

```diff
@@ -3,8 +3,6 @@ on:
   schedule:
     - cron: '20 22 * * *'  # 10:20pm everyday
   push:
-    branches:
-      - main
```

Comment:
Wouldn't this run the workflow on all pushes to all branches?  Inc PR branches?

Reply:
It's hard to explain, the way it's working in main is the `workflow_dispatch` is handling both the PRs and merge to main separately and in order.

Reply:
If you want to queue up some changes and pile up a handful of PRs to watch it in realtime I've found that to be useful!

Reply:
Well, this PR pushed a new `:37` image :smile: 

https://github.com/ublue-os/bluefin/actions/runs/4821382578/jobs/8587179489#step:9:71

Reply:
I'm checking in `main` as I may be misunderstanding something